lib/pipeline.js

```diff
@@ -135,6 +135,82 @@ Pipeline.prototype = {
   },
 
   /**
+   * Bind to a streams events, and re-emit them on the pipeline
+   * @param {Stream} stream
+   * @param {String|Array} events
+   * @returns {Pipeline}
+   * @chainable
+   */
+  bind( stream, events ) {
```

Comment:
Nah, that'd prevent unbinding of events after a stream's been removed from the pipeline and such. Also, there might be a case where you'd want to bind to a stream's events where the stream is not a direct part of the pipeline in question (i.e. nested pipelines).